client/state/rewind/retention/test/reducer.js

```diff
@@ -39,25 +39,25 @@ describe( 'updateBackupRetentionRequestStatus', () => {
 					type: JETPACK_BACKUP_RETENTION_UPDATE_RESET,
 				}
 			)
-		).toEqual( BACKUP_RETENTION_UPDATE_REQUEST.UNSUBMITTED );
+		).toEqual( { requestStatus: BACKUP_RETENTION_UPDATE_REQUEST.UNSUBMITTED } );
 	} );
 
 	test( 'should return PENDING status when retention update request action is initiated', () => {
 		// lets update backup retention for siteId:123 to 7 days.
 		expect(
 			updateBackupRetentionRequestStatus( undefined, updateBackupRetention( 123, 7 ) )
-		).toEqual( BACKUP_RETENTION_UPDATE_REQUEST.PENDING );
+		).toEqual( { requestStatus: BACKUP_RETENTION_UPDATE_REQUEST.PENDING } );
 	} );
 
-	test( 'should return SUCCESS status when retention update request action gets completed successfully', () => {
+	test( '1should return SUCCESS status when retention update request action gets completed successfully', () => {
```

Comment:
```suggestion
	test( 'should return SUCCESS status when retention update request action gets completed successfully', () => {
```
Sorry for the nit :)

Reply:
Thanks! I added that `1` unintentionally 😅